### ISSUE TITLE
Docker install fails on Vagrant

### ISSUE THREAD

**Issue by loshkovskyi** [CiscoCloud/microservices-infrastructure#627]
Here is the issue:
```
==> default: TASK: [docker | install latest device-mapper-libs] **************************** 
==> default: changed: [default]
==> default: 
==> default: TASK: [docker | install docker repo] ****************************************** 
==> default: changed: [default]
==> default: 
==> default: TASK: [docker | install docker packages] ************************************** 
==> default: failed: [default] => (item=docker-engine-1.7.0,docker-selinux) => {"changed": true, "item": "docker-engine-1.7.0,docker-selinux", "rc": 1, "results": ["docker-selinux-1.6.0-11.0.1.el7.centos.x86_64 providing docker-selinux is already installed", "Loaded plugins: fastestmirror\nLoading mirror speeds from cached hostfile\n * epel: mirror.steadfast.net\nResolving Dependencies\n--> Running transaction check\n---> Package docker-engine.x86_64 0:1.7.0-1.el7.centos will be installed\n--> Processing Dependency:   [...]   Processing Conflict: docker-engine-1.7.0-1.el7.centos.x86_64 conflicts docker\n--> Processing Conflict: docker-engine-1.7.0-1.el7.centos.x86_64 conflicts docker-io\n--> Finished Dependency Resolution\n You could try using --skip-broken to work around the problem\n You could try running: rpm -Va --nofiles --nodigest\n"]}
==> default: msg: Error: docker-engine conflicts with docker-1.6.2-14.el7.centos.x86_64
```

Vagrant box comes with pre-installed Docker:
```
-bash-4.2# rpm -qa|grep docker
docker-1.6.0-11.0.1.el7.centos.x86_64
docker-selinux-1.6.0-11.0.1.el7.centos.x86_64
```
Since we install Docker 1.7.0 when provisioning VM, the deployment fails.

**Comment by stevendborrelli** [comment]
Thanks for finding this. Let me take a look.

**Comment by ryane** [comment]
There was a new version of the vagrant image published the other day that includes the fixes in #612. You want to make sure you get version 0.5.

```shell
$ vagrant box list
CiscoCloud/microservices-infrastructure (virtualbox, 0.5)
```

I found I actually had to delete old versions first (using `vagrant box remove`) before Vagrant would pull down the latest image.

**Comment by loshkovskyi** [comment]
@ryane thank you, it worked out. Indeed, there is a newer Docker inside. I think I will close this issue. Would be great to have such option in Vagrantfile to specify a specific version of the box.
